app/statistics/server/functions/get.js

```diff
@@ -154,5 +156,34 @@ statistics.get = function _getStatistics() {
 	statistics.uniqueOSOfYesterday = Sessions.getUniqueOSOfYesterday();
 	statistics.uniqueOSOfLastMonth = Sessions.getUniqueOSOfLastMonth();
 
+	if (Apps.isEnabled()) {
+		const installedApps = Apps.getManager().get();
+		const activeApps = Apps.getManager().get({ enabled: true });
+		statistics.apps = {
+			totalInstalled: installedApps.length,
+			totalActive: activeApps.length,
+			installedAppsList: installedApps.map((app) => ({
```

Comment:
do we really need send this info? it does not go against what we say about statistics reporting (no user information is sent) but still I wonder why do we need this data? specially the app name?

Reply:
This would give us a way to understand what apps are in what state in the servers, therefore we could have info like "what % of this particular app is not enabled on the servers it's installed in".

Regarding the name.... maybe we could take it out, I guess? 🤔

Reply:
Wait... we don’t need the names or info.  We only want to see how heavily integrations are used.  Count should be good enough.  Including names I think crosses line of what people would be comfortable sharing.  I wouldn’t want to share this either

Reply:
I think most useful here is:

Count of types of integration, and how many have a script attached

Reply:
Ok, so remove list, only identify active vs inactive count?

Reply:
Ok, so remove list, only identify active vs inactive count?

Reply:
> Including names I think crosses line of what people would be comfortable sharing

agreed! that was exactly my feeling.

I'll do the changes then..